src/components/Kv/KvButton.vue

```diff
@@ -81,6 +81,36 @@ export default {
 		box-shadow: $button-secondary-shadow darken($kiva-accent-red, 10%);
 	}
 
+	&.classic {
+		border: rem-calc(3) solid $kiva-green;
+		box-shadow: none;
+		border-radius: rem-calc(20);
```

Comment:
Can we get away with `class="classic rounded"` and remove this diverging border-radius?

Reply:
I do like the idea of a rounded class that can be applied separately alongside a color and "hollow" classes...